### DISCUSSION TITLE
2.3 -> 2.4: using modified /opt/so/saltstack/local/pillar/global.sls from 2.3?

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#11494] [General]
Hi,
With 2.3 reaching EOL in April 2024 (https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html) it's time to think about going from 2.3 to 2.4.
As per https://docs.securityonion.net/en/2.4/appendix.html the suggested way is to install 2.4 instead of trying to upgrade an existing 2.3 to 2.4.

Over time I've made quite some changes to /opt/so/saltstack/local/pillar/global.sls that I'd like to also have available in 2.4.

My question: After installing 2.4 in a distributed environment (i.e. separate sensor node, index node, several IDHs, manager node) is it possible to simply replace the stock /opt/so/saltstack/local/pillar/global.sls that comes with 2.4 with the one I've got on 2.3?

Thanks much in advance for any clue,
-ewald

**Comment by dougburks** [comment]
In 2.4, you should avoid modifying config files in the filesystem and instead use the config interface:
https://docs.securityonion.net/en/2.4/administration.html#configuration
